docs/docusaurus/versioned_docs/version-0.18/resources/get_support.md

```diff
@@ -4,9 +4,26 @@ title: 'Get support'
 description: Learn what resources are available to help you resolve issues with GX Cloud and GX OSS.
 ---
 
-Great Expectations (GX) relies on an active user community for its continued success. Input from users like you helps GX and the community build its collective knowledge and improve GX Cloud and GX OSS. Knowing how and where to seek support will help you get the answers you need to resolve issues.
+## Where to get support
 
-## Try solving your issue first
+### GX Cloud support
+
+GX Cloud customers can reach the GX support team through the following channels.
+
+- Sending an email to support@greatexpectations.io
+- Creating a new topic on the [GX Cloud Discourse forum](https://discourse.greatexpectations.io/c/cloud-support/17).
+- Reaching out in the [GX Cloud Slack community channel](https://greatexpectationstalk.slack.com/archives/C051D941XAL).
+
+### GX OSS support
+
+If you're using GX OSS only, you can get support via the community.
+
+- Chat with community members about general issues and seek solutions on the [GX Slack community channel](https://greatexpectationstalk.slack.com/archives/CUTCNHN82).
+- If you've found a bug related to OSS, open an issue in our [GX GitHub repository](https://github.com/great-expectations/great_expectations).
+
+## How to effectively get support
```

Comment:
![image](https://github.com/great-expectations/great_expectations/assets/104205257/8c7cad7f-29de-4c3b-8d15-b3441347225c)

Add one or two sentences that describe the content that follows the heading and its purpose. Alternatively, remove the existing H2 heading and promote the H3 headings. For help with heading structure, see [Heading and title format](https://developers.google.com/style/headings#heading-and-title-format).